tests/testthat/test-seasonal_onset.R

```diff
@@ -20,16 +20,16 @@ test_that("The growth rate models converge", {
     time_interval = "day"
   )
 
-  # Calculate AEDSEO with a 3-day window
-  aedseo_poisson <- aedseo(
+  # Calculate seasonal_onset with a 3-day window
+  tsd_poisson <- seasonal_onset(
     tsd = tsd_data_poisson,
     k = 3,
     level = 0.95,
     family = "poisson",
     disease_threshold = 20,
     na_fraction_allowed = 0.2
   )
-  aedseo_quasipoisson <- aedseo(
+  tsd_quasipoisson <- seasonal_onset(
     tsd = tsd_data_nbinom,
```

Comment:
This is likely out of scope of this PR, but it is unclear if the underlying distribution of the test data is "quasipoisson" or "negative binomial".